src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,79 @@
+package view;
+
+import model.player.Player;
+import model.ladder.LadderRow;
+import model.player.PlayerResults;
+
+import java.util.List;
+import java.util.Map;
+
+public class OutputView {
+    private static final int COLUMN_LENGTH = 5;
+
+    private void outputPlayer(List<Player> playerInventory) {
+        for (Player player : playerInventory) {
+            String playerName = player.getName();
+            System.out.printf("%" + COLUMN_LENGTH + "s", playerName);
+            System.out.printf(" ");
+        }
+    }
+
+    private void outputLadderRow(List<Boolean> points) {
+        System.out.println();
+        System.out.print("    |");
+        for (int i = 0; i < points.size(); i++) {
+            outputLadderColumn(points.get(i));
+        }
+    }
+
+    private void outputLadderColumn(Boolean bool) {
+        if (bool == true) System.out.print("-----|");
```

Comment:
컨벤션 상으로 
```suggestion
        if (bool == true){
            System.out.print("-----|");
        }
```
로 바뀌면 좋을 것 같아요!

Reply:
if 문을 쓸 때는 뒤에 한 줄의 코드만 있더라도 일정한 형식을 지키는 게 좋군요!!
수정완료 하였습니다!!